requirements.txt

```diff
@@ -34,7 +34,7 @@ psycopg2-binary==2.7.4
 pycparser==2.13
 Pygments==1.6rc1
 pyOpenSSL==17.5.0
-pysam==0.9.1.4
+pysam==0.15.3
```

Comment:
Have you tested this? I got an error during installation:
```
(refinery-platform) vagrant@refinery:/vagrant/refinery$ pip install pysam==0.15.3
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Collecting pysam==0.15.3
  Using cached https://files.pythonhosted.org/packages/15/e7/2dab8bb0ac739555e69586f1492f0ff6bc4a1f8312992a83001d3deb77ac/pysam-0.15.3.tar.gz
    ERROR: Command errored out with exit status 1:
     command: /home/vagrant/.virtualenvs/refinery-platform/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-jQZFUB/pysam/setup.py'"'"'; __file__='"'"'/tmp/pip-install-jQZFUB/pysam/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-jQZFUB/pysam/pip-egg-info
         cwd: /tmp/pip-install-jQZFUB/pysam/
    Complete output (131 lines):
    # pysam: no cython available - using pre-compiled C
    # pysam: htslib mode is shared
    # pysam: HTSLIB_CONFIGURE_OPTIONS=None
    # pysam: (sysconfig) CC=x86_64-linux-gnu-gcc -pthread
    # pysam: (sysconfig) CFLAGS=-fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -Wdate-time -D_FORTIFY_SOURCE=2 -g -fstack-protector-strong -Wformat -Werror=format-security
    # pysam: (sysconfig) LDFLAGS=-Wl,-Bsymbolic-functions -Wl,-z,relro
    checking for gcc... x86_64-linux-gnu-gcc -pthread
    checking whether the C compiler works... yes
    checking for C compiler default output file name... a.out
    checking for suffix of executables...
    checking whether we are cross compiling... no
    checking for suffix of object files... o
    checking whether we are using the GNU C compiler... yes
    checking whether x86_64-linux-gnu-gcc -pthread accepts -g... yes
    checking for x86_64-linux-gnu-gcc -pthread option to accept ISO C89... none needed
    checking for ranlib... ranlib
    checking for grep that handles long lines and -e... /bin/grep
    checking for C compiler warning flags... unknown
    checking for special C compiler options needed for large files... no
    checking for _FILE_OFFSET_BITS value needed for large files... no
    checking for _LARGEFILE_SOURCE value needed for large files... no
    checking shared library type for unknown-Linux... plain .so
    checking how to run the C preprocessor... x86_64-linux-gnu-gcc -pthread -E
    checking for egrep... /bin/grep -E
    checking for ANSI C header files... yes
    checking for sys/types.h... yes
    checking for sys/stat.h... yes
    checking for stdlib.h... yes
    checking for string.h... yes
    checking for memory.h... yes
    checking for strings.h... yes
    checking for inttypes.h... yes
    checking for stdint.h... yes
    checking for unistd.h... yes
    checking for stdlib.h... (cached) yes
    checking for unistd.h... (cached) yes
    checking for sys/param.h... yes
    checking for getpagesize... yes
    checking for working mmap... yes
    checking for gmtime_r... yes
    checking for fsync... yes
    checking for drand48... yes
    checking whether fdatasync is declared... yes
    checking for fdatasync... yes
    checking for library containing log... -lm
    checking for zlib.h... yes
    checking for inflate in -lz... yes
    checking for library containing recv... none required
    checking for bzlib.h... no
    checking for BZ2_bzBuffToBuffCompress in -lbz2... no
    configure: error: libbzip2 development files not found

    The CRAM format may use bzip2 compression, which is implemented in HTSlib
    by using compression routines from libbzip2 <http://www.bzip.org/>.

    Building HTSlib requires libbzip2 development files to be installed on the
    build machine; you may need to ensure a package such as libbz2-dev (on Debian
    or Ubuntu Linux) or bzip2-devel (on RPM-based Linux distributions or Cygwin)
    is installed.

    Either configure with --disable-bz2 (which will make some CRAM files
    produced elsewhere unreadable) or resolve this error to build HTSlib.
    checking for gcc... x86_64-linux-gnu-gcc -pthread
    checking whether the C compiler works... yes
    checking for C compiler default output file name... a.out
    checking for suffix of executables...
    checking whether we are cross compiling... no
    checking for suffix of object files... o
    checking whether we are using the GNU C compiler... yes
    checking whether x86_64-linux-gnu-gcc -pthread accepts -g... yes
    checking for x86_64-linux-gnu-gcc -pthread option to accept ISO C89... none needed
    checking for ranlib... ranlib
    checking for grep that handles long lines and -e... /bin/grep
    checking for C compiler warning flags... unknown
    checking for special C compiler options needed for large files... no
    checking for _FILE_OFFSET_BITS value needed for large files... no
    checking for _LARGEFILE_SOURCE value needed for large files... no
    checking shared library type for unknown-Linux... plain .so
    checking how to run the C preprocessor... x86_64-linux-gnu-gcc -pthread -E
    checking for egrep... /bin/grep -E
    checking for ANSI C header files... yes
    checking for sys/types.h... yes
    checking for sys/stat.h... yes
    checking for stdlib.h... yes
    checking for string.h... yes
    checking for memory.h... yes
    checking for strings.h... yes
    checking for inttypes.h... yes
    checking for stdint.h... yes
    checking for unistd.h... yes
    checking for stdlib.h... (cached) yes
    checking for unistd.h... (cached) yes
    checking for sys/param.h... yes
    checking for getpagesize... yes
    checking for working mmap... yes
    checking for gmtime_r... yes
    checking for fsync... yes
    checking for drand48... yes
    checking whether fdatasync is declared... yes
    checking for fdatasync... yes
    checking for library containing log... -lm
    checking for zlib.h... yes
    checking for inflate in -lz... yes
    checking for library containing recv... none required
    checking for bzlib.h... no
    checking for BZ2_bzBuffToBuffCompress in -lbz2... no
    configure: error: libbzip2 development files not found

    The CRAM format may use bzip2 compression, which is implemented in HTSlib
    by using compression routines from libbzip2 <http://www.bzip.org/>.

    Building HTSlib requires libbzip2 development files to be installed on the
    build machine; you may need to ensure a package such as libbz2-dev (on Debian
    or Ubuntu Linux) or bzip2-devel (on RPM-based Linux distributions or Cygwin)
    is installed.

    Either configure with --disable-bz2 (which will make some CRAM files
    produced elsewhere unreadable) or resolve this error to build HTSlib.
    make: ./version.sh: Command not found
    make: ./version.sh: Command not found
    config.mk:2: *** Resolve configure error first.  Stop.
    # pysam: htslib configure options: None
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-jQZFUB/pysam/setup.py", line 241, in <module>
        htslib_make_options = run_make_print_config()
      File "/tmp/pip-install-jQZFUB/pysam/setup.py", line 68, in run_make_print_config
        stdout = subprocess.check_output(["make", "-s", "print-config"])
      File "/usr/lib/python2.7/subprocess.py", line 574, in check_output
        raise CalledProcessError(retcode, cmd, output=output)
    subprocess.CalledProcessError: Command '['make', '-s', 'print-config']' returned non-zero exit status 2
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

Reply:
I did try this out but it worked for me.  I have seen this before though (going through the Python3 business) and I think it better then to just wait for that since I have something to ameliorate this.